common/src/app/common/data.cljc

```diff
@@ -256,6 +256,15 @@
 (defn zip [col1 col2]
   (map vector col1 col2))
 
+(defn zip-long
```

Comment:
zip-long is a very bad name, because it refers to `long` native number type

Reply:
changed